File: 백준/Bronze/4388. 받아올림/받아올림.cc
#include <algorithm>
#include <iostream>
using namespace std;
int main(void) {
    string a, b;
    int carry, cnt;
    while (1) {
        cin >> a >> b;
        carry = 0, cnt = 0;
        if (a == "0" && b == "0") break;
        if (a.size() < b.size()) swap(a, b);
        while (a.size() != b.size()) b = "0" + b;
        for (int i = a.size() - 1; i >= 0; i--) {
            carry = (carry + (a[i] - 48) + (b[i] - 48)) / 10;
            if (carry > 0) cnt += 1;
        }
        cout << cnt << "\n";
    }
    return 0;
}

File: 백준/Bronze/4388. 받아올림/받아올림.py
import sys
input=lambda:sys.stdin.readline().rstrip()
while True:
    a,b=input().split()
    if a==b=='0': break
    if len(a)<len(b): a,b=b,a;
    while len(a)!=len(b): b='0'+b
    carry,cnt=0,0
    for i in range(len(a)-1,-1,-1):
        carry=(carry+int(a[i])+int(b[i]))//10
        if carry>0: cnt+=1
    print(cnt)